src/TSTLErrors.ts

```diff
@@ -18,6 +18,22 @@ export class TSTLErrors {
     public static ForbiddenForIn = (node: ts.Node) =>
         new TranspileError(`Iterating over arrays with 'for ... in' is not allowed.`, node);
 
+    public static ForbiddenLuaTableSetExpression = (node: ts.Node) => new TranspileError(
+        `A '@LuaTable' object's 'set()' method can only be used as a Statement, not an Expression.`,
+        node);
+
+    public static ForbiddenLuaTableNonDeclaration = (node: ts.Node) =>
+        new TranspileError(`Classes with the '@luaTable' decorator must be declared.`, node);
+
+    public static InvalidExtendsLuaTable = (node: ts.Node) =>
+        new TranspileError(`Cannot extend classes with the decorator '@luaTable'.`, node);
+
+    public static InvalidInstanceOfLuaTable = (node: ts.Node) =>
+        new TranspileError(`The instanceof operator cannot be used with a '@luaTable' class.`, node);
+
+    public static ForbiddenLuaTableUseException = (node: ts.Node, description: string) =>
```

Comment:
By convention we have node as the last parameter. I am not sure about having this function like this at all since it does nothing but passing the same parameters to the transpileError.

I would prefix the description with "Invalid @luaTable usage: " to give it some purpose in life.